custom_components/tuya_local/devices/agl_ultramagic_lock.yaml

```diff
@@ -113,15 +71,66 @@ secondary_entities:
             value: High
   - entity: switch
     category: config
-    name: Auto close
+    name: Allow Intercom Unlock
```

Comment:
Are these configuration switches?  The naming "approve_intercom" etc suggests that they are to be used to unlock the door in response to a "request_intercom" signal, which is what the lock entity is able to handle if the simpler "lock" dp is left out.